### ISSUE TITLE
Transcriber - Check warning (TODO)

### ISSUE THREAD

**Issue by jovanovic-djo** [jovanovic-djo/dp-tv-show-transcript#8]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

**Comment by jovanovic-djo** [comment]
fixed lol
